Vehicle.h

```diff
@@ -5,20 +5,16 @@
 
 struct Vehicle
 {
-    Vehicle(const std::string& n) : name(n) { }
+    Vehicle(const std::string& n);
     
-    virtual void setSpeed(int s)
-    {
-        std::cout << "\nHighway is adjusting " << name << "'s speed to " << s << " mph" << std::endl;
-        speed = s;
-    }
+    virtual ~Vehicle(); // FIXME these are implementations.  move them to the CPP file
+    Vehicle(const Vehicle&);
+    Vehicle& operator = (const Vehicle&);
     
-    virtual void tryToEvade()
-    {
-        std::cout << name << ": you'll never take me alive, ya dirty coppers!" << std::endl;
-        setSpeed(100);
-    }
-protected:
+    virtual void setSpeed(int s);
+    virtual void tryToEvade();
+
     int speed = 0;
```

Comment:
- [ ] speed is private originally.   leave it private. 
- [ ] re-read the instructions for how to make Vehicle work with HighwayPatrol